libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-dv/defaults.parm

```diff
@@ -1,630 +1,467 @@
-SERVO3_TRIM 995
-SERVO4_FUNCTION 0
-SERVO4_MAX 2000
-SERVO4_MIN 995
-SERVO4_REVERSED 0
-SERVO4_TRIM 995
-SERVO5_FUNCTION 0
-SERVO5_MAX 1900
-SERVO5_MIN 1100
-SERVO5_REVERSED 0
-SERVO5_TRIM 1500
-SERVO6_FUNCTION 0
-SERVO6_MAX 1900
-SERVO6_MIN 1000
-SERVO6_REVERSED 0
-SERVO6_TRIM 1500
-SERVO7_FUNCTION 0
-SERVO7_MAX 1900
-SERVO7_MIN 1100
-SERVO7_REVERSED 0
-SERVO7_TRIM 1500
-SERVO8_FUNCTION 0
-SERVO8_MAX 1900
-SERVO8_MIN 1000
-SERVO8_REVERSED 0
-SERVO8_TRIM 1500
-SERVO9_FUNCTION 33
-SERVO9_MAX 2000
-SERVO9_MIN 1000
-SERVO9_REVERSED 0
-SERVO9_TRIM 1000
-
-SERVO_BLH_AUTO 1
-SERVO_BLH_BDMASK 3840
-SERVO_BLH_OTYPE 5
-SERVO_BLH_POLES 42
-SERVO_BLH_TRATE 100
-SERVO_DSHOT_RATE 1
-SERVO_FTW_POLES 42
-SERVO_RATE 400
-SERVO_SBUS_RATE 50
-
-SR0_ADSB 0
-SR0_EXTRA1 0
-SR0_EXTRA2 0
-SR0_EXTRA3 0
-SR0_EXT_STAT 0
-SR0_PARAMS 0
-SR0_POSITION 0
-SR0_RAW_CTRL 0
-SR0_RAW_SENS 0
-SR0_RC_CHAN 0
-SR1_ADSB 0
-SR1_EXTRA1 0
-SR1_EXTRA2 0
-SR1_EXTRA3 0
-SR1_EXT_STAT 0
-SR1_PARAMS 0
-SR1_POSITION 0
-SR1_RAW_CTRL 0
-SR1_RAW_SENS 0
-SR1_RC_CHAN 0
-SR2_ADSB 0
-SR2_EXTRA1 10
-SR2_EXTRA2 10
-SR2_EXTRA3 2
-SR2_EXT_STAT 2
-SR2_PARAMS 0
-SR2_POSITION 10
-SR2_RAW_CTRL 0
-SR2_RAW_SENS 2
-SR2_RC_CHAN 2
-SR3_ADSB 0
-SR3_EXTRA1 0
-SR3_EXTRA2 0
-SR3_EXTRA3 0
-SR3_EXT_STAT 0
-SR3_PARAMS 0
-SR3_POSITION 0
-SR3_RAW_CTRL 0
-SR3_RAW_SENS 0
-SR3_RC_CHAN 0
-SR4_ADSB 0
-SR4_EXTRA1 0
-SR4_EXTRA2 0
-SR4_EXTRA3 0
-SR4_EXT_STAT 0
-SR4_PARAMS 0
-SR4_POSITION 0
-SR4_RAW_CTRL 0
-SR4_RAW_SENS 0
-SR4_RC_CHAN 0
-SRTL_ACCURACY 2
-SRTL_OPTIONS 0
-SRTL_POINTS 500
-
-SUPER_SIMPLE 0
-SURFTRAK_MODE 0
-SURFTRAK_TC 1
-TCAL_ENABLED 0
-TELEM_DELAY 0
-TERRAIN_OFS_MAX 0
-THR_DZ 35
-TKOFF_RPM_MAX 0
-TKOFF_RPM_MIN 1200
-TKOFF_SLEW_TIME 2
-TKOFF_THR_MAX 0.9
-WPNAV_ACCEL 350
-WPNAV_ACCEL_C 250
-WPNAV_ACCEL_Z 150
-WPNAV_JERK 5
-WPNAV_RADIUS 100
-WPNAV_RFND_USE 0
-WPNAV_SPEED 600
-WPNAV_SPEED_DN 300
-WPNAV_SPEED_UP 500
-WPNAV_TER_MARGIN 10
-WP_NAVALT_MIN 0
-WP_YAW_BEHAVIOR 2
-WVANE_ENABLE 0
+
+AHRS_EKF_TYPE,3
+AHRS_GPS_USE,2
+AHRS_ORIENTATION,2
+AHRS_TRIM_X,0
+AHRS_TRIM_Y,0
+AHRS_TRIM_Z,0
+
+ANGLE_MAX,2500
+
+ARMING_CHECK,1
+ARMING_MAGTHRESH,200
+
+ATC_ACCEL_P_MAX,50000
+ATC_ACCEL_R_MAX,50000
+ATC_ACCEL_Y_MAX,13000
+ATC_ANG_LIM_TC,2
+ATC_ANG_PIT_P,12
+ATC_ANG_RLL_P,11.5
+ATC_ANG_YAW_P,6.5
+ATC_ANGLE_BOOST,1
+ATC_INPUT_TC,0.2
+ATC_RAT_PIT_D,0.0045
+ATC_RAT_PIT_FLTD,10
+ATC_RAT_PIT_FLTT,10
+ATC_RAT_PIT_I,0.095
+ATC_RAT_PIT_IMAX,0.5
+ATC_RAT_PIT_P,0.095
+ATC_RAT_RLL_D,0.0038
+ATC_RAT_RLL_FLTD,10
+ATC_RAT_RLL_FLTT,10
+ATC_RAT_RLL_I,0.1
+ATC_RAT_RLL_IMAX,0.5
+ATC_RAT_RLL_P,0.1
+ATC_RAT_YAW_FLTD,10
+ATC_RAT_YAW_FLTE,2.2
+ATC_RAT_YAW_FLTT,10
+ATC_RAT_YAW_I,0.07
+ATC_RAT_YAW_IMAX,0.5
+ATC_RAT_YAW_P,0.7
+ATC_RATE_FF_ENAB,1
+ATC_RATE_P_MAX,180
+ATC_RATE_R_MAX,180
+ATC_RATE_Y_MAX,180
+ATC_SLEW_YAW,6000
+ATC_THR_G_BOOST,0
+ATC_THR_MIX_MAN,0.1
+ATC_THR_MIX_MAX,0.7
+ATC_THR_MIX_MIN,0.1
+
+AVOID_ACCEL_MAX,3
+AVOID_ALT_MIN,0
+AVOID_ANGLE_MAX,1000
+AVOID_BACKUP_DZ,0.1
+AVOID_BACKUP_SPD,0.75
+AVOID_BEHAVE,0
+AVOID_DIST_MAX,5
+AVOID_ENABLE,3

```

Comment:
It's not my area of expertise, but I think that may be set to 1:
The parameter is a bitfield, so 3 means UseFence AND UseProximitySensor.
![image](https://github.com/user-attachments/assets/87a18e31-7f3d-412f-aada-99253a165faf)

Reply:
This was set when we were doing Object avoidance tests with Dijkstra's and Bendy Ruler. It would use the Geo Fence Inclusions and Exclusions in the UseFence part. Also we prepared it to test with the front lidar as proximity sensor but the Lidar use on the Kobra was unreliable at that time. Until further testing, the Lidars are disabled and the OA_ TYPE param is set to 0.

Reply:
When you write that OA_TYPE = 0, is that the default? because you are not explicitly setting it in this list @DaanAT 

Shouldn't we remove all AVOID params if we are not using them?

Reply:
As OA_TYPE = 0 is default I mark this as solved, and as long as it does not affect flight behavior it doesn't matter. 

I don't think we have reached a final conclusion on the actual practical use of the front facing rangefinder.

Reply:
If it is default it should be removed from this list and added when it is actually changed.

Reply:
OA_TYPE is not in the parameterlist as it is 0 by default.